test/pretty.test.js

```diff
@@ -380,3 +381,66 @@ test('does not throw error when enabled with stream specified', function (t) {
   pino({prettyPrint: true}, process.stdout)
   t.end()
 })
+
+test('CLI: --localTime option', function (t) {
+  t.plan(5)
+  var now = Date.now()
+  var log = {
+    level: 30,
+    msg: 'hello world',
+    time: now,
+    pid: process.pid,
+    hostname: hostname,
+    v: 1
+  }
+  var child = spawn('node', [path.join(__dirname, '../bin.js'), '--localTime'], { stdio: 'pipe' })
+  child.stdin.end(JSON.stringify(log))
+
+  var line = ''
+  child.stdout.on('data', (data) => {
+    line += data.toString('binary')
+  })
+
+  child.on('close', function () {
+    line = line.slice(0, -1 * (os.EOL).length)
+    var localTime = line.slice(line.indexOf('[') + 1, line.indexOf(']'))
+    t.ok(line.match(/.*hello world$/), 'end of line matches')
+    t.ok(line.match(/(?!^)INFO.*/), 'includes level')
+    t.ok(line.indexOf('' + process.pid) > 0, 'includes pid')
+    t.ok(line.indexOf('' + hostname) > 0, 'includes hostname')
+    t.ok(Date.parse(localTime) === parseInt(now), 'localTime <-> UTC match')
+    return line
+  })
+})
+
+test('CLI: --dateFormat option', function (t) {
+  t.plan(5)
+  var now = Date.now()
+  var log = {
+    level: 30,
+    msg: 'hello world',
+    time: now,
+    pid: process.pid,
+    hostname: hostname,
+    v: 1
+  }
+  var child = spawn('node', [path.join(__dirname, '../bin.js'), '--dateFormat', 'YYYY/MM/DDThh,mm,ss_SSSZ'], { stdio: 'pipe' })
+  child.stdin.end(JSON.stringify(log))
+
+  var line = ''
+  child.stdout.on('data', (data) => {
+    line += data.toString('binary')
+  })
+
+  child.on('close', function () {
+    line = line.slice(0, -1 * (os.EOL).length)
+    var formatDate = line.slice(line.indexOf('[') + 1, line.indexOf(']'))
+    var toISODate = formatDate.replace(/\//g, '-').replace(/,/g, ':').replace(/_/g, '.')
+    t.ok(line.match(/.*hello world$/), 'end of line matches')
+    t.ok(line.match(/(?!^)INFO.*/), 'includes level')
+    t.ok(line.indexOf('' + process.pid) > 0, 'includes pid')
+    t.ok(line.indexOf('' + hostname) > 0, 'includes hostname')
+    t.ok(Date.parse(toISODate) === parseInt(now), 'custDateFormat <-> UTC match')
```

Comment:
This needs to be checked using an interval. These numbers are not assured to be identical on a CI system, as they are dependent on what else is running.

Reply:
how can i check custom dateformat or local-time without passing epo

Reply:
how can I check transformed dateformat or local-time is correct? 
this is my thought, I pass time from msg (log.time = log.msg), then i can check the datestring

Reply:
How about 

```
t.ok(Date.parse(toISODate) > parseInt(now) - 2000)
t.ok(Date.parse(toISODate) < parseInt(now) + 2000)
```

Reply:
sure, I misunderstand your words. 
by the way, is there any suggestions/tricks for comparing floating-point ?